scripts/amd/gemm/tune_gemm.py

```diff
@@ -404,10 +449,10 @@ def tune_gemm_config(M, N, K, col_a, col_b, dtype_a, dtype_b, dtype_c, init_type
     thread_pool = multiprocessing.Pool(processes=num_threads)
     tasks = []
     idx = 0
-    df_prof = [pd.read_csv(f"results-{i}.csv") for i in range(jobs)]
+    df_prof = [pd.read_csv(f"results-{generated_kernel_name(M, N, K, i)}.csv") for i in range(jobs)]
```

Comment:
this one should also be reverted to use `i` instead of the kernel_name with absolute path

Reply:
Thanks, fixed.